src/clients/add-on_auto.rb

```diff
@@ -56,9 +56,25 @@ def main
       Builtins.y2debug("param=%1", @param)
 
       if @func == "Import"
-        @ret = AddOnProduct.Import(
-          Convert.convert(@param, :from => "map", :to => "map <string, any>")
-        )
+        add_on_products = @param["add_on_products"] || []
+        count = 0
+        # Checking needed values
+        add_on_products.reject! do |product|
+          count += 1
+          unless product.has_key?("media_url")
+            # Report missing media_url entry in the AutoYaST configuration file
+            # TRANSLATORS: The placeholder points to the location in the AutoYaST configuration file.
+            error_string = format(_("Error in the AutoYaST <add_on> section.\n" \
+              "Missing mandatory <media_url> value in the %d. product definition."),
```

Comment:
Suggestion:
"Missing mandatory <media_url> value at index %d in the <add_on_products> definition."